### PR TITLE
Play 2.5 support

### LINKED ISSUE #47: Solve Play 2.5 version conflicts
Version 2.0.0 supports both Play 2.4 and Play 2.5 but in plugins.sbt Play 2.4 is used.
That couse version conflicts during sbt's dependency resolving:

```
[warn]  * com.typesafe.play:play_2.11:2.5.18 is selected over 2.4.0
[warn]      +- com.typesafe.play:play-cache_2.11:2.5.18           (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-jdbc-api_2.11:2.5.18        (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-logback_2.11:2.5.18         (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-ws_2.11:2.5.18              (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-server_2.11:2.5.18          (depends on 2.5.18)
[warn]      +- org.julienrf:play-jsmessages_2.11:2.0.0            (depends on 2.4.0)
```

### PR DESCRIPTION
**Pull request by fugafree**
It fixes #47.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,7 +14,7 @@ lazy val jsmessages = project
   .settings(commonSettings: _*)
   .settings(
     name := "play-jsmessages",
-    crossScalaVersions := Seq("2.10.5", "2.11.6"),
+    crossScalaVersions := Seq("2.11.12"),
```
Comment (julienrf) [inline build.sbt]: You can even remove `crossScalaVersions`.

**Comment by julienrf** [comment]
Add the following file to the [.travis.yml](https://github.com/julienrf/play-jsmessages/blob/2.1.x/.travis.yml) file:

~~~
scala:
  - 2.11.12
~~~

(like we [currently](https://github.com/julienrf/play-jsmessages/blob/master/.travis.yml) have for Play 2.6, which supports both 2.11 and 2.12)

**Comment by julienrf** [comment]
Excellent, thanks! Would you mind squashing your commits?

**Comment by fugafree** [comment]
It's okey.

**Comment by julienrf** [comment]
Thanks!

**Comment by fugafree** [comment]
Me too! Would you please release it?

**Pull request merged**